pkg/api/image/schema.go

```diff
@@ -17,10 +17,19 @@ func RegisterSchema(scaled *config.Scaled, server *server.Server, options config
 		Customize: func(s *types.APISchema) {
 			s.Formatter = Formatter
 			s.ResourceActions = map[string]schemas.Action{
-				actionUpload: {},
+				operationUpload: {},
 			}
 			s.ActionHandlers = map[string]http.Handler{
-				actionUpload: UploadActionHandler{
+				operationUpload: ImageHandler{
+					httpClient:                  http.Client{},
+					Images:                      scaled.HarvesterFactory.Harvesterhci().V1beta1().VirtualMachineImage(),
+					ImageCache:                  scaled.HarvesterFactory.Harvesterhci().V1beta1().VirtualMachineImage().Cache(),
+					BackingImageDataSources:     scaled.LonghornFactory.Longhorn().V1beta1().BackingImageDataSource(),
+					BackingImageDataSourceCache: scaled.LonghornFactory.Longhorn().V1beta1().BackingImageDataSource().Cache(),
+				},
+			}
+			s.LinkHandlers = map[string]http.Handler{
```

Comment:
May we add some comments here for the difference between `ActionHandlers` and `LinkHandlers`? Thank you.

Reply:
OK, I will add some comments for `ActionHandlers` and `LinkHandlers`!

Reply:
Already add some comments for `ActionHandlers` and `LinkHandlers`.
Please check again, thanks!

Reply:
Perhaps we can create just one ImageHandler instance for Link and Action.

Reply:
good suggestion! Done.